content/en/docs/ambient/extensibility/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Extensibility
+description: Demonstrates how to extend ambient mesh behavior.
+weight: 45
+keywords: [extensibility,WebAssembly,Wasm]
+test: n/a
+---
```

Comment:
@craigbox thoughts on structure?  I could see us have a tasks section under ambient and extensibility under tasks.

Reply:
imo, mirroring the sidecar doc structure would be the most logical so there's consistency between the two sets.

Reply:
> @craigbox thoughts on structure? I could see us have a tasks section under ambient and extensibility under tasks.

I have moved the extensibility folder under ambient/usage. Let me know if that is okay.